web/src/components/ShelterListItem.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react'
+
+import { ShelterModel } from 'api-client'
+
+import CleanLink from './CleanLink'
+import ShelterInformation from './ShelterInformation'
+
+type Props = {
+  shelter: ShelterModel
+}
+
+const ShelterListItem = ({ shelter }: Props): ReactElement => (
+  <CleanLink to={shelter.id.toString()}>
+    <ShelterInformation shelter={shelter} />
+    {/* TODO IGAPP-944: Add button "Mehr Informationen"/"Anschauen" */}
```

Comment:
@f1sh1918 if you want you could do this. not urgent though as it works for now.

Reply:
added button, even there should be a link but you can not render a link within a link, so i misused a button
Shall the whole card be clickable and the button?

Reply:
No, fine like this. Is more of a visual thing, the whole list item should be clickable. So well done :+1: